npm/react/docs/recipes.md

```diff
@@ -32,6 +32,30 @@ See example repo [bahmutov/try-cra-with-unit-test](https://github.com/bahmutov/t
 
 **Tip:** `plugins/react-scripts` is just loading `plugins/cra-v3`.
 
+### With custom webpack preprocessor options
+
+To modify the `preprocessorOptions` that are passed into the `webpackPreprocessor` you can pass a function as `preprocessorOptionsFinal` option into the `react-scripts` plugin, e.g.
+
+```js
+// cypress/plugins/index.js
+module.exports = (on, config) => {
+  require('@cypress/react/plugins/react-scripts')(on, config, {
+      preprocessorOptionsFinal(preprocessorOptions) {
```

Comment:
Related to naming. I think that we can make it a little bit simpler, we can do the same as next.js doing in config. And this API should be available for all the plugins 

```js
 webpack: (config) => {
    config.module.rules.push({
      test: /\.md$/,
      use: "raw-loader",
    });

    return config;
  },
```